src/main/java/io/github/_4drian3d/authmevelocity/lastserver/configuration/Configuration.java

```diff
@@ -0,0 +1,24 @@
+package io.github._4drian3d.authmevelocity.lastserver.configuration;
+
+import org.spongepowered.configurate.objectmapping.ConfigSerializable;
+import org.spongepowered.configurate.objectmapping.meta.Comment;
+
+import java.util.List;
+
+@ConfigSerializable
+@SuppressWarnings({"CanBeFinal", "FieldMayBeFinal"})
+public final class Configuration {
+    @Comment("Require players to have the lastserver.use permission?")
+    private Boolean requirePermission = false;
+
+    @Comment("A list of servers to not save as a player's last server")
+    private List<String> excludedServers = List.of("login", "spawn");
+
+    public Boolean getRequirePermission() {

```

Comment:
It is not necessary to use the wrapper, just use the primitive type
```suggestion
    public boolean getRequirePermission() {
```